### PR TITLE
[GUI] MasternodeWidget-Wizard bugfixes

### PR DESCRIPTION
**Pull request by random-zebra**
This PR fixes some issue with the masternode GUI interaction.

**1)**  __MasternodeWizard: lock collateral output after MN creation.__
The masternode configuration file is only read during init, and then the `COutPoint`s relative to the collateral of each entry is locked for spending. Thus, after creating a MN controller with the wizard, the collateral remains unlocked until the wallet is restarted, and therefore could be potentially spent or staked.

**2)** __Enforce required depth for mn collaterals__
The masternode collateral transaction requires at least 15 confirmations before the mn broadcast can be considered valid. Properly inform the user after creating a controller with the wizard, and when trying to start a masternode not yet confirmed.

**3)** __MasternodeWidget: unlock collateral output coin after MN deletion__
Mirrors (1). Unlock the collateral utxo when the masternode is deleted, otherwise it remains locked until wallet restart.

**4)** __MasterNodeWizardDialog: fix RegTestNet default port__
RegTest was using the fixed port value of TestNet instead of the default 51476.

### CODE REVIEW

File: src/qt/pivx/masternodeswidget.cpp [hunk 1]
```diff
@@ -133,7 +133,8 @@ MasterNodesWidget::MasterNodesWidget(PIVXGUI *parent) :
     connect(ui->btnAboutController, &OptionButton::clicked, [this](){window->openFAQ(10);});
 }
 
-void MasterNodesWidget::showEvent(QShowEvent *event){
+void MasterNodesWidget::showEvent(QShowEvent *event)
```
Comment (Fuzzbawls) [inline src/qt/pivx/masternodeswidget.cpp]: Could you extract out these styling changes to their own commit?

Comment (random-zebra) [reply src/qt/pivx/masternodeswidget.cpp]: sure, can do.

Comment (random-zebra) [reply src/qt/pivx/masternodeswidget.cpp]: done
